frontend/modules/account/operations.js

```diff
@@ -105,20 +109,78 @@ function createNewBitcoinAccount() {
     };
 }
 
+function startLis() {
+    return async (dispatch, getState) => {
+        const {
+            privacyMode,
+            termsMode,
+            lisStatus,
+            lightningID,
+        } = getState().account;
+        if (
+            privacyMode !== types.PRIVACY_MODE.EXTENDED
+            || termsMode !== types.TERMS_MODE.ACCEPTED
```

Comment:
How it could work without terms accepted?

Reply:
Upon first login of already registered user after update, before he accepts terms in new flow

Reply:
Check on start lis takes place on initAccount before actual modal flow. Just after he accepts terms and choose extended it will be up